### PR TITLE
Create a script to do most things automatically

### PR DESCRIPTION
**Pull request by vrutkovs**
Replace run instructions with a simple script

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+set -eu
+
+USERNAME=$1
```
Comment (mtnbikenc) [inline start.sh:9]: Running without arguments just gives
```
./start.sh: line 4: $1: unbound variable
```

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+set -eu
```
Comment (mtnbikenc) [inline start.sh]: Adding `x` option helps when commands fail so you know which command was being run.

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  exit 1
+fi
+
+echo "Removing existing artifacts"
+rm -rvf install-config.yml .openshift_install_state.json .openshift_install.log *.ign || true
+
+echo
+echo "Fetching installer"
+podman pull ${INSTALLER_IMAGE}
+podman run -rm -ti ${INSTALLER_IMAGE} version
```
Comment (mtnbikenc) [inline start.sh]: Need `sudo` on these lines too.

**Review by mtnbikenc — COMMENTED** [review]
A few issues.

File: start.sh (hunk 1 above)
Comment (vrutkovs) [reply start.sh:9]: Fixed

File: start.sh (hunk 2 above)
Comment (vrutkovs) [reply start.sh]: Yeah, agreed

File: start.sh (hunk 3 above)
Comment (vrutkovs) [reply start.sh]: this now can be changed via updating `PODMAN`

File: start.sh [hunk 4]
```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+set -eux
+
+if [ -z $1 ]; then
```
Comment (mtnbikenc) [inline start.sh]: $1 is still unbound if an argument is not given.  Maybe,
```bash
if (( $# != 1 )); then
```

File: start.sh [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+rm -rvf install-config.yml .openshift_install_state.json .openshift_install.log *.ign || true
+
+echo
+echo "Fetching installer"
+${PODMAN} pull ${INSTALLER_IMAGE}
+${PODMAN} run -rm -ti ${INSTALLER_IMAGE} version
+
+echo
+echo "Creating bootstrap.ign"
+${PODMAN} run --rm \
```
Comment (mtnbikenc) [inline start.sh]: I still need to add `--privileged` here and later `podman run` commands.

Comment (vrutkovs) [reply start.sh]: Added `PODMAN_RUN` and `PODMAN_PULL` for this

File: README.md [hunk 6]
```diff
@@ -4,86 +4,42 @@
...
+* GCP account credentials in a json
+* SSH private key
 
 # Prepare GCP creds
-Prepare a folder which consists the following files (see `example-injected` folder):
+Put your pull secret in `pull_secret.json`
+
+Prepare "injected/" directory which contains the following files (see `example-injected` folder):
 * `gce.json` - this is the JSON which GCP uses to authenticate. `libra.pem` should work
 * `ssh-privatekey` - private key which would be used to access instances
```
Comment (gpei) [inline README.md]: Do we need to move this tip  ``libra.pem` should work ` to the next line  * `ssh-privatekey` ?

Comment (vrutkovs) [reply README.md]: Oh, correct. Fixed

**Comment by mtnbikenc** [comment]
The script works well.  Might want to add a note that `--privileged` may need to be added to the `PODMAN_RUN` command.  At least I need it on my system or I get the error below.

```
FATAL Error executing openshift-install: failed to setup logging hook: failed to open log file: open .openshift_install.log: permission denied
```

**Comment by vrutkovs** [comment]
@gpei mentioned `--user $(id -u)` worked after a few tries, I'll check out if that fixes it

**Pull request merged**